app/domain/authentication/constraints/any_constraint.rb

```diff
@@ -0,0 +1,19 @@
+module Authentication
+  module Constraints
```

Comment:
should this module go under the `Common` module?

Reply:
It can, but I like it in a separate folder since they are very isolated classes, with very common structure.
IMO it makes sense to keep them isolated in a different module/folder.

Reply:
i'm fine with whichever decision you take here. I was suggesting to have the structure `Authentication -> Common -> Constraints`. But again, i;m ok with the current structure

Reply:
@abrahamko can i get an indication that you read this? if you still decide not to do it it's ok but i want to verify that you considered it.

Reply:
Thanks for the suggestion, and I decided to keep it in the `authentication1 folder.